src/arrays.js

```diff
@@ -1,61 +1,93 @@
 const getNthElement = (index, array) => {
-  // your code here
+  if (index < array.length) {
+    return array[index];
+  }
+  if (index >= array.length) {
+    return array[index - array.length];
+  }
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.toString();
 };
 
 const csvStringToArray = string => {
-  // your code here
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  return array.concat(element);
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  return array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+  return numbers.map(String);
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+  const newArray = [];
+
+  for (let i = 0; i < strings.length; i++) {
+    newArray.push(strings[i].toUpperCase());
+  }
+  return newArray;
 };
+// Could do with refactoring to make more concise ^^ (use a higher order function) //
 
 const reverseWordsInArray = strings => {
-  // your code here
+  const reverseWords = strings.map(word =>
+    word
+      .split('')
+      .reverse()
+      .join('')
+  );
+  return reverseWords;
 };
 
 const onlyEven = numbers => {
-  // your code here
+  let evenNumbers = [];
+  for(let i = 0; i < numbers.length; i++) {
+    if (numbers[i] % 2 === 0) {
+      evenNumbers.push(numbers[i]);
+    }
+  }
+  return evenNumbers;
```

Comment:
This does work, but it can be made more concise using a filter method. Most of the time arrays are manipulated using either `filter`, `map`, `forEach` or `sort` depending on what you are trying to do. In this case you are trying to remove all values that dont fit a particular criteria (those that have a remainder after dividing by 2): 
```suggestion
  return numbers.filter(number => number % 2 === 0);
```